metricbeat-yml-script.py

```diff
@@ -181,8 +186,9 @@ def _add_aws_shipping_data():
 
 
 def _get_tags_value(aws_namespace):
-    if "aws/" in aws_namespace.lower():
-        service_name = aws_namespace.split("/")[1].lower()
+    aws_namespace_lower = aws_namespace.lower()
+    if "aws/" in aws_namespace_lower:
```

Comment:
Why so general? isn't that always a prefix/suffix?

Reply:
Nope. You can insert a custom namespace, which doesn't require a prefix.
Made this function to cover as many popular services as it can, and I think that for our current needs it's enough.